### DISCUSSION TITLE
[Issue Forms] Render markdown in body

### DISCUSSION THREAD

**Discussion by issc29** [community/community#5631] [Projects and Issues]
According to the documentation:

> You can use a markdown element to display Markdown in your form that provides extra context to the user, but is not submitted.

There are use cases where it would be useful to submit the markdown so that it's rendered in the Issue body. An example would be adding links to useful content and putting some text that makes the Issue more readable after it is submitted

**Comment by Phoenix616** [comment]
This would be really useful for cases where logs/config files are asked for as they could be automatically hidden behind a summary around a textarea's output. A possible way to implement this would be to provide a per-element display template attribute.

**Comment by nijel** [comment]
Having separate toggle for rendering fields would be useful. In our case, I'd like to avoid including (some) required checkboxes - the user has to toggle them to submit the issue, but they are not useful after that.

**Comment by SirSpidey** [comment]
Pulling together related requests for renderable Markdown:https://github.com/orgs/community/discussions/63402#discussioncomment-10341167
